src/views/Play/CustomGames.tsx

```diff
@@ -74,6 +74,12 @@ export function CustomGames(): JSX.Element {
     const canvas: HTMLCanvasElement = React.useMemo(() => allocateCanvasOrError(), []);
     const seekgraph = React.useRef<SeekGraph>();
 
+    const [seekGraphVisible, setSeekGraphVisible] = React.useState(false);
```

Comment:
Maybe should do a `usePreference` here instead of `useState` so we remember their setting?

Reply:
Doh yes!